addon/mixins/dom.js

```diff
@@ -184,7 +184,7 @@ export default Mixin.create({
       /*
        * Attach the listener and cache the listener for teardown
        */
-      element.on(eventName, coalescedCallback);
+      element.addEventListener(eventName, coalescedCallback);
```

Comment:
I believe this should indicate that it is passive (thats the only way we get down this path):

```js
element.addEventListener(eventName, coalescedCallback, { passive: true });
```

And I think the corresponding `element.removeEventListener` needs to pass `{ passive: true }` also (inside `_removeCoalescedEventListener`).